app/controllers/v1/actors_controller.rb

```diff
@@ -2,6 +2,65 @@ class V1::ActorsController < ApplicationController
   skip_before_action :authorize_request
   before_action :set_actor, only: [:show, :update, :destroy]
 
+  swagger_controller :actors, "Actors Management"
+
+  swagger_api :index do
+    summary "Fetches all Actors"
+    notes "This lists all the active actors"
+    response :ok, "Success", :Actor
+    response :unauthorized, "The actor does not have permission to perform this action"
+    response :invalid_token, "The provided API token is invalid"
+    response :forbidden, "This resource cannot be accessed"
+  end
+
+  swagger_api :show do
+    summary "Fetches a specific Actor"
+    notes "This fetches the Actor matching the given id"
+    param :path, :id, :integer, :required, "Actor Id"
+    response :ok, "Success", :Actor
+    response :not_found, "Actor not found"
+    response :unauthorized, "The actor does not have permission to perform this action"
+    response :invalid_token, "The provided API token is invalid"
+    response :forbidden, "This resource cannot be accessed"
+  end
+
+  swagger_api :create do
+    summary "Creates an Actor"
+    notes "This creates a new Actor"
+    param :form, :fullname, :string, :required, "The full name of the actor"
+    param :form, :email, :string, :required, "The Actor's email address"
+    response :ok, "Success", :Actor
```

Comment:
Also, if time allows, the text strings could be defined as constants somewhere else. This could simplify reuse (I see some strings are the same in other files) and simplify translations in a future.

Reply:
@taliaga Done it for the responses in the controllers. For defining strings in different files, it will be done in the future if we need to work multiple translations ^^

Reply:
Thanks, so much better. Ok, let's forget about strings in different files for now. But one last thing... couldn't this `add_common_params` method be defined in a shared module? a.l.a `V1::ControllerUtils::add_common_params(api)` to avoid the redefinition of it in each file. What do you think? 😸

Reply:
@taliaga I wanted to define a common set of parameters in the top level Application controller, but for some reason it didn't seem to register the parameters correctly, it would just ignore them. For now I'm defining the common ones per controller, also in case I change something and want slightly different responses in some of the busier controllers like Cluster and NGSISubs